tests/utils/fixtures-generator.js

```diff
@@ -0,0 +1,19 @@
+const { random } = require('lodash');
+const FixturesGenerator = require('fixtures-generator');
+const SCHEMAS = require('./schemas');
```

Comment:
Separate installed libs from internal libs, position deconstructing objects imported at the end of the block
```suggestion
const FixturesGenerator = require('fixtures-generator');
const { random } = require('lodash');

const SCHEMAS = require('./schemas');
```